include/server/SC_WorldOptions.h

```diff
@@ -104,8 +104,8 @@ SCSYNTH_DLLEXPORT_C void SetPrintFunc(PrintFunc func);
 SCSYNTH_DLLEXPORT_C struct World* World_New(struct WorldOptions *inOptions);
 SCSYNTH_DLLEXPORT_C void World_Cleanup(struct World *inWorld, bool unload_plugins = false);
 SCSYNTH_DLLEXPORT_C void World_NonRealTimeSynthesis(struct World *inWorld, struct WorldOptions *inOptions);
-SCSYNTH_DLLEXPORT_C int World_OpenUDP(struct World *inWorld, int inPort);
-SCSYNTH_DLLEXPORT_C int World_OpenTCP(struct World *inWorld, int inPort, int inMaxConnections, int inBacklog);
+SCSYNTH_DLLEXPORT_C int World_OpenUDP(struct World *inWorld, const char *bindTo, int inPort);
```

Comment:
btw, this is a change to the public API. it will break code for people using libscsynth via an FFI (iirc the haskel bindings are doing this)